web/js/index.js

```diff
@@ -0,0 +1,271 @@
+
+import {
+    PerspectiveCamera,
+    Scene,
+    Fog,
+    AmbientLight,
+    WebGLRenderer,
+    DirectionalLight,
+    PMREMGenerator,
+    sRGBEncoding,
+    Box3,
+    Vector3
+} from 'three';
+import { WEBGL } from 'three/examples/jsm/WebGL.js';
+import { GLTFLoader } from 'three/examples/jsm/loaders/GLTFLoader.js';
+import { TrackballControls } from 'three/examples/jsm/controls/TrackballControls.js';
+
+(() => {
```

Comment:
The use of an IIFE here seems a bit 'odd' and not something I'd expect. I'm not a fan of them (especially not for an entire file) because they tend to lead to a large indent, and also a habit of 'run on functions' which can be hard to contextualize when reading the code.

However I'm not familiar with Rollup (we tend to use Webpack, but I've no objections to using rollup here) - so is there some reason Rollup wants an IIFE?

Reply:
Removed in new push.